webdriver-spec.html

```diff
@@ -504,6 +505,68 @@
 </section> <!-- /Conformance -->
 
 <section>
+<h2>Introduction</h2>
+
+<p><i>This section is non-normative</i>
+
+<section>
+<h3>Design Notes</h3>
+
+<p>This specification attempts to follow a number of design goals:
+
+<section>
+<h4>Compatibility with existing Selenium WebDriver functionality</h4>
+
+<p>This specification is derived from the popular
+ <a href=https://seleniumhq.org>Selenium WebDriver</a> browser
+ automation framework. Selenium is a long-lived project, and due to
+ its age and breadth of use it has a wide range of expected
+ functionality. This specification uses these expectations to inform
+ its design. Where improvements or clarifications have been made, they
+ have been made with care to allow existing users Selenium WebDriver
+ to avoid unexpected breakages.
+</section>
+
+<section>
+<h4>Suitability for use end-user browser automation needs</h4>
+
+<p>It is expected that the largest group of users of this
+ specification are going to be software developers and testers writing
+ automated tests and other tooling, such as monitoring or load
+ testing, that relies on automating a browser. As such, care has been
+ taken to provide commands that simplify common tasks such
+ as <a data-lt="Element Send Keys">typing into</a>
+ and <a data-lt="Element Click">clicking</a> elements.
+
+</section>
+
+<section>
+<h4>Use by other specifications</h4>
+
+<p>WebDriver aims to allow other specifications
+ to <a data-lt="extension commands">extend the spec</a> to support new
+ functionality, and to allow conformance tests that cannot be
+ implemented entirely in
+ <a href=https://tc39.github.io/ecma262/>Ecmascript</a> to be written
+ in a vendor-neutral way.
+
+</section>
+
+<section>
+<h4>Compliance with other specifications</h4>
+
+<p>This specification relies as much as possible on behaviors and
+ functionality expressed in other specifications. Wherever possible,
+ this specification avoids defining new functionality unless this
+ conflicts with its other design goals.
```

Comment:
I think this entire paragraph can be dropped because it doesn’t say anything non-obvious.